lib/index.js

```diff
@@ -57,42 +57,63 @@ bedrock.events.on('bedrock.configure', function applyConfig() {
   }
 });
 
+function handleConfigLoadError(e, {configType}) {
+  // Only rethrow raw error in development to avoid leaking sensitive info.
+  if(process.env.NODE_ENV == 'development') {
```

Comment:
Style: triple equals.

```suggestion
  if(process.env.NODE_ENV === 'development') {
```

I also don't know if it's possible for a deployment to have `development` set in some way ... accidentally on purpose, or whatever.